src/analysis/retail/demonhunter/vengeance/guide/useVdhFeatureFlag.ts

```diff
@@ -1,17 +1,10 @@
-import { useState } from 'react';
+import useSessionFeatureFlag from 'interface/useSessionFeatureFlag';
 
-const useVdhFeatureFlag = (featureFlag: string = '', featureFlagDefault?: boolean) => {
-  const sessionFeatureFlagSetting = window.sessionStorage?.getItem(
-    // This will filter out if featureFlag is an empty string
-    ['wowa-vdh-ff', featureFlag].filter((it) => it).join('-'),
+const useVdhFeatureFlag = (featureFlag: string, featureFlagDefault: boolean = false) => {
```

Comment:
Does 'vdh' stand for Vengeance Demon Hunter? Is this supposed to be spec specific?

Reply:
This one was intended to be a wrapper around the usual "useSessionFeatureFlag" hook so that I could have VDH specific flags. You could use "useSessionFeatureFlag" without creating a wrapper hook